GPU/GPUTracking/Base/cuda/CMakeLists.txt

```diff
@@ -114,13 +114,27 @@ if(ALIGPU_BUILD_TYPE STREQUAL "O2")
     ${MODULE}
     SOURCES ${SRCS}
     PUBLIC_LINK_LIBRARIES O2::GPUTracking O2::ITStrackingCUDA
+    PRIVATE_LINK_LIBRARIES ONNXRuntime::ONNXRuntime
     PRIVATE_INCLUDE_DIRECTORIES
       ${CMAKE_SOURCE_DIR}/Detectors/Base/src
       ${CMAKE_SOURCE_DIR}/Detectors/TRD/base/src
       ${CMAKE_SOURCE_DIR}/DataFormats/Reconstruction/src
       ${CMAKE_CURRENT_SOURCE_DIR}
     TARGETVARNAME targetName)
 
+  message("Compile definitions for ONNX runtime (CUDA):")
+  message(STATUS "ORT_ROCM_BUILD: ${ORT_ROCM_BUILD}")
+  message(STATUS "ORT_CUDA_BUILD: ${ORT_CUDA_BUILD}")
+  message(STATUS "ORT_MIGRAPHX_BUILD: ${ORT_MIGRAPHX_BUILD}")
+  message(STATUS "ORT_TENSORRT_BUILD: ${ORT_TENSORRT_BUILD}")
+
+  target_compile_definitions(${targetName} PRIVATE
+    GPUCA_HAS_ONNX=1
+    ORT_ROCM_BUILD=$<BOOL:${ORT_ROCM_BUILD}>
+    ORT_CUDA_BUILD=$<BOOL:${ORT_CUDA_BUILD}>
```

Comment:
do we need ROCM and MIGRAPHX compile definitions for CUDA? I assume we can remove them here, and add them only for the ROCm one?